bindgen/bindgen.bzl

```diff
@@ -16,7 +16,19 @@
 load("//rust:rust.bzl", "rust_library")
 
 # buildifier: disable=bzl-visibility
-load("//rust/private:utils.bzl", "find_toolchain", "get_libs_for_static_executable")
+load("//rust/private:utils.bzl", "find_toolchain", "get_preferred_artifact")
+
+def _get_libs_for_static_executable(dep):
```

Comment:
Pls add a todo for me to use the more robust logic you're adding in rustc.bzl also here, through a reasonable API.